docs/SlavaUkraini/examples/overlays/index.md

```diff
@@ -0,0 +1,253 @@
+
+
+#### A bit of control over the video
+
+If you read the API documentation, you'll notice that the `L.VideoOverlay` class does not have a `play()` or `pause()` method.
+
+For this, the `getElement()` method of the video overlay is useful. It returns the [`HTMLVideoElement`](https://developer.mozilla.org/docs/Web/API/HTMLImageElement) (which inherits from [`HTMLMediaElement`](https://developer.mozilla.org/docs/Web/API/HTMLMediaElement)) for the overlay - and that has methods like `play()` and `pause()`, e.g.
+
+```
+videoOverlay.getElement().pause();
+```
+
+This allows us to build custom interfaces. For example, we can build a small subclass of `L.Control` to play/pause this video overlay once it's loaded:
+
+```
+videoOverlay.on('load', function () {
+	var MyPauseControl = L.Control.extend({
+		onAdd: function() {
+			var button = L.DomUtil.create('button');
+			button.title = 'Pause';
+			button.innerHTML = '<span aria-hidden="true">⏸</span>';
+			L.DomEvent.on(button, 'click', function () {
+				videoOverlay.getElement().pause();
+			});
+			return button;
+		}
+	});
+	var MyPlayControl = L.Control.extend({
+		onAdd: function() {
+			var button = L.DomUtil.create('button');
+			button.title = 'Play';
+			button.innerHTML = '<span aria-hidden="true">▶️</span>';
+			L.DomEvent.on(button, 'click', function () {
+				videoOverlay.getElement().play();
+			});
+			return button;
+		}
+	});
+
+	var pauseControl = (new MyPauseControl()).addTo(map);
+	var playControl = (new MyPlayControl()).addTo(map);
+});
+```
+
+{% include frame.html url="example-video.html" %}
+
+### `SVGOverlay`
+
+`L.SVGOverlay` is used to load, display and provide DOM access to an SVG file over specific bounds of the map. 
+
+To add an SVG overlay [`L.SVGOverlay`](/reference.html#svgoverlay) use this:
+
+```
+var svgOverlay = L.svgOverlay(SVGElement, svgElementBounds, options);
+```
+
+It instantiates an image overlay object given an SVG element and the geographical bounds it is tied to. A viewBox attribute is required on the SVG element to zoom in and out properly.
+
+#### Creating an SVG element
+
+Let's create an SVG element:
+
+```
+var svgElement = document.createElementNS('http://www.w3.org/2000/svg', 'svg');
+svgElement.setAttribute('xmlns', 'http://www.w3.org/2000/svg');
+svgElement.setAttribute('viewBox', '0 0 200 200');
+svgElement.innerHTML = '<rect width="200" height="200"/><rect x="75" y="23" width="50" height="50" style="fill:red"/><rect x="75" y="123" width="50" height="50" style="fill:#0013ff"/>';
+```
+
+Alternatively, you can create the SVG element in HTML code:
+
+```
+<svg id="svg" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 200 200"><rect width="200" height="200"/><rect x="75" y="23" width="50" height="50" style="fill:red"/><rect x="75" y="123" width="50" height="50" style="fill:#0013ff"/></svg>
+```
+
+And choose this SVG element by using a querySelector:
+
+```
+var svgElement = document.querySelector('#svg');
+```
+
+#### Creating the SVG Overlay
```

Comment:
The corresponding VideoOverlay section's heading is "Adding the video overlay", I suggest changing this heading to "Adding the SVG overlay".